CrossPlatformDesktopProject/CrossPlatformDesktopProject/Commands/CycleObstaclesT.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CrossPlatformDesktopProject.Commands
+{
+    class CycleObstaclesT : ICommand
+    {
+        private Game1 myGame;
+        private Sprint2ListStorage sL;
+        public CycleObstaclesT(Game1 game, Sprint2ListStorage sL)
+        {
+            myGame = game;
+            this.sL = sL;
+        }
+        public void Execute() => myGame.obstacle = Sprint2ListStorage.obstacles[sL.lastObstacleIndex()];
```

Comment:
This shouldn't have to so much about the implementation details of Sprint2ListStorage.

I'll refactor this a bit.